<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color: qlineargradient(spread:pad, x1:0.064, y1:0.107727, x2:0.869, y2:0.79, stop:0 rgba(23, 26, 33, 255), stop:0.806147 rgba(42, 71, 94, 255));}</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>420</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(199, 213, 224);</string>
    </property>
    <property name="text">
     <string>Choose your scan type</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dicomUploadButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>270</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Verdana&quot;;
color: rgb(227, 243, 255);
border-radius: 5px;
background-color: rgb(102, 192, 244);</string>
    </property>
    <property name="text">
     <string>Dicom/Dcm Scan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mhdUploadButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>270</y>
      <width>270</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Verdana&quot;;
color: rgb(227, 243, 255);
border-radius: 5px;
background-color: rgb(102, 192, 244);</string>
    </property>
    <property name="text">
     <string>Mhd Scan</string>
    </property>
   </widget>
   <widget class="QLabel" name="mhdIcon">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>256</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../assets/mhd.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="mhdIcon_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>350</y>
      <width>72</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../assets/dicom.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
